lib/testing/src/lib/core/pages/login-sso.page.ts

```diff
@@ -53,6 +72,31 @@ export class LoginSSOPage {
         await browser.waitForAngular();
     }
 
+    async loginBasicAuth(username, password): Promise<void> {
+        await this.goToLoginPage();
+
+        await LocalStorageUtil.clearStorage();
```

Comment:
clearing the localStorage at every login makes it very difficult to automate some scenarios that require keeping the storage and testing things using the localStorage with 2 different users (like preserving sorting, pagination per user for example)